### PR TITLE
Updated src/ - Clean npm lint test

### PR DESCRIPTION
**Pull request by Frenzy-code**
xd lol the problem was with the wrapper

### CODE REVIEW

**Comment by Frenzy-code** [comment]
I edited fibRoute.ts to handle edge cases and type handling.

**Comment by Frenzy-code** [comment]
It used to call `fib()` on non-numbers, now it handles that and returns 401 and 402 HTTP error code.

File: src/fibRoute.ts [hunk 1]
```diff
@@ -1,16 +1,31 @@
 // Endpoint for querying the fibonacci numbers
 
-const fibonacci = require("./fib");
+import { fibonacci } from "./fib";
```
Comment (nejail) [inline src/fibRoute.ts:3]: Good way of fixing the error of the import that made the lint run

**Review by nejail — COMMENTED** [review]
Overall, the code looks good, and with the lint test it passes every test needed, should be merged to main!

**Comment by Frenzy-code** [comment]
Yotsuba Koiwai
